seatunnel-translation/seatunnel-translation-base/src/main/java/org/apache/seatunnel/translation/source/CoordinatedSource.java

```diff
@@ -165,7 +165,12 @@ public void run(Collector<T> collector) throws Exception {
                                         while (flag.get()) {
                                             try {
                                                 reader.pollNext(collector);
-                                                Thread.sleep(SLEEP_TIME_INTERVAL);
+                                                if (collector.isEmptyThisPollNext()) {
+                                                    Thread.sleep(100);
+                                                } else {
+                                                    collector.resetEmptyThisPollNext();
+                                                    Thread.sleep(0L);
```

Comment:
Shall we add some comment just like https://github.com/apache/seatunnel/pull/5695/files#diff-e09fdadf2d09ab76a701fa16a47e4e899418490cc7bb4d8223a2d44266a02ab9 ?

Reply:
> Shall we add some comment just like https://github.com/apache/seatunnel/pull/5695/files#diff-e09fdadf2d09ab76a701fa16a47e4e899418490cc7bb4d8223a2d44266a02ab9 ?

Yes, A comment is required.